lib/core/app/jobs/ros/platform_event_process_job.rb

```diff
@@ -2,6 +2,9 @@
 
 module Ros
   class PlatformEventProcessJob < Ros::ApplicationJob
+    def perform(*params)
```

Comment:
@rjayroach For now I think its better to just override the perform method for these two jobs and create a jira ticket to address the operations one by one.

Reply:
https://perxtechnologies.atlassian.net/browse/PW-2185